src/hooks/useABTest/useABTest.js

```
@@ -0,0 +1,27 @@
+import { useMemo, useEffect } from 'react';
+import { hash } from './hash';
+
+const logToPendo = ({ testName, variantName }) => {
+  if (window.pendo) {
```

Should we throw an error somewhere if no `testName` is supplied to the hook?

---

We can and probably should. Definitely a required parameter.